src/webgl/interactions/hitArea/HitArea.ts

```diff
@@ -0,0 +1,36 @@
+import { AllFederatedEventMap, Container, IHitArea } from 'pixi.js'
+import RenderObject from '../../RenderObject'
+
+export default class HitArea<T extends IHitArea = IHitArea> extends RenderObject<Container> {
```

Comment:
### HitArea
- base class can be used on its own or extended to define a particular shape (circle, line, rect, etc.)